Library/Formula/gtk-mac-integration.rb

```diff
@@ -0,0 +1,156 @@
+class GtkMacIntegration < Formula
+  homepage "https://wiki.gnome.org/Projects/GTK%2B/OSX"
+  url "http://ftp.gnome.org/pub/gnome/sources/gtk-mac-integration/2.0/gtk-mac-integration-2.0.7.tar.xz"
+  sha256 "e6de70da740b452f18c113a7ebb6772936a0fb9873137514090849820e296eb7"
+
+  depends_on "pkg-config" => :build
+  depends_on "xz" => :build
```

Comment:
You can scrap this I think, unless it's run-time, which presumably it isn't? Homebrew seems to automatically call an ` xz ` dep for ` tar.xz ` formula, which I believe is by design.